iroha_p2p/src/lib.rs

```diff
@@ -0,0 +1,9 @@
+//! This module provides a network layer for holding of persistent connections
+//! between blockchain nodes. Also, it provides for simple use of encryption
+//! to hide the data flowing between nodes.
+
+mod message;
+mod network;
+mod peer;
+
+pub use network::Network;
```

Comment:
May be also reexport public message types (for which there are handlers for Network) here?